packets2/willtopicresp_test.go

```diff
@@ -0,0 +1,62 @@
+package packets2
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	pkts "github.com/energomonitor/bisquitt/packets"
+)
+
+func TestWillTopicRespConstructor(t *testing.T) {
+	assert := assert.New(t)
+
+	pkt := NewWillTopicResp(RC_SUCCESS)
```

Comment:
Same question as in `WillMsgTest`: Any reason why the code is changed from `RC_COGNESTION` used in the same test in `packets1`?

Reply:
Same as above. Fixed.